xbmc/network/EventServer.cpp

```diff
@@ -186,7 +182,7 @@ void CEventServer::Run()
     CLog::Log(LOGERROR, "ES: Invalid port range specified %d, defaulting to 10", port_range);
     port_range = 10;
   }
-  if (!m_pSocket->Bind(any_addr, m_iPort, port_range))
+  if (!m_pSocket->Bind(!CSettings::GetInstance().GetBool(CSettings::SETTING_SERVICES_ESALLINTERFACES), m_iPort, port_range))
```

Comment:
yes. security (no, I am neither paranoid, nor a security fanatic). expose as less you can by default to the bad guys waiting to make a botnet out of your app :)

most users will never need a non-local eventserver and jsonserver